src/components/Home/form/form.css

```diff
@@ -1,3 +1,30 @@
-#submit {
+#addhead {
```

Comment:
- [ ] To make your styling to the next level, please use `classes` instead of `ids`!